tests/pytests/test_issues.py

```diff
@@ -560,4 +560,14 @@ def test_mod_4207(env):
   conn.execute_command('HSET', 'address:4', 'country', 'Israel')
 
   env.expect('FT.SEARCH', 'idx1', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:4'])
-  env.expect('FT.SEARCH', 'idx2', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:3'])
\ No newline at end of file
+  env.expect('FT.SEARCH', 'idx2', '*', 'NOCONTENT').equal([3, 'address:1', 'address:2', 'address:3'])
+
+def test_mod_4255(env):
+  env.expect('FT.CREATE', 'idx', 'ON', 'HASH', 'SCHEMA', 'test', 'NUMERIC').equal('OK')
```

Comment:
The test only covers the `revisionId` of the numeric index. We should have a test that also validate the `gcMarker`.